.github/workflows/ci.yml

```diff
@@ -35,6 +35,26 @@ jobs:
           do_not_skip: '["workflow_dispatch", "schedule", "push"]'
           concurrent_skipping: false
 
+  scan-container:
```

Comment:
Because our image is light-weight (alpine & binary, eventually just scratch & binary), I'm not sure the value of scanning the image. If dependabot can update the base image dependency when an update is available, then this basically handles the security of the base image.

Reply:
true, it's not necessary right now. For me it's just another layer of protection. If in future we add entrypoint/whatnot scripts or install os packages we won't think about scanning the image ^.^ (not really sure if we really have the need to install packages or scripts).

Let me remove it for now. 👍